browser/brave_ads/sources.gni

```diff
@@ -73,8 +73,6 @@ if (is_android) {
     "//brave/browser/brave_ads/android/brave_ads_native_helper.h",
     "//brave/browser/brave_ads/background_helper/background_helper_android.cc",
     "//brave/browser/brave_ads/background_helper/background_helper_android.h",
-    "//brave/browser/brave_ads/brave_ads_host_android.cc",
```

Comment:
It also needed to move `brave/browser/brave_ads/brave_ads_host.cc` to `brave_browser_brave_ads_sources` section

Reply:
Thanks for catching this!